backend/src/types/__init__.py

```diff
@@ -0,0 +1,3 @@
+from typing import List, Tuple
+
+bbox_type = Tuple[int, int, int, int]
```

Comment:
```suggestion
BBoxType = Tuple[int, int, int, int]
```
https://docs.python.org/3/library/typing.html#type-aliases